docs/topic/graph-matching/graph-match.md

```diff
@@ -1,73 +1,78 @@
 author: accelsao
 
- **匹配** 或是 **独立边集** 是一张图中没有公共边的集合。
-在二分图中求匹配等价于网路流问题。
+**Matching** or **independent set of edges** is a set with no common edges in a graph. Finding a match in a bipartite graph is equivalent to the network flow problem.
 
-图匹配算法是信息学竞赛中常用的算法，总体分为最大匹配以及最大权匹配，先从二分图开始介绍，在进一步提出一般图的作法。
+Graph matching algorithm is a commonly used algorithm in informatics competitions. It can be generally divided into maximum matching and maximum weight matching. We will first introduce the bipartite graph, and then propose solutions for general graph.
```

Comment:
```suggestion
Graph matching algorithm are commonly used in informatics competitions. It can be generally divided into maximum matching and maximum weight matching. We will first introduce the bipartite graph, and then propose solutions for general graphs.
```